### ISSUE TITLE
V11 Massive amount of warnings and minor slowdowns every time pathfinding is triggered

### ISSUE THREAD

**Issue by apoapostolov** [manuelVo/foundryvtt-drag-ruler#286]
```
You are accessing CONFIG.Canvas.losbackend, which is deprecated. Use CONFIG.Canvas.polygonBackends.sight instead.
Deprecated since Version 11
Backwards-compatible support will be removed in Version 13
```

**Comment by rexmortus** [comment]
My guess is this  has to do with a module that is calling `canvas.walls.checkCollision`

This has been deprecated in favor of: `CONFIG.Canvas.polygonBackends.move.testCollision`

The methods work a bit differently. With canvas.walls.checkCollision you provide it a `Ray` object between the positions of the first and second tokens, whereas with `CONFIG.Canvas.polygonBackends.move.testCollision` you simply provided it two objects with the x,y coordinates of each token along with some options `type` and `mode`.

Let's say you have 2 token objects, you could check if LOS between them was obstructed like this:

```
let losBlocked = CONFIG.Canvas.polygonBackends.move.testCollision(token1.center, token2.center, { type: "move", mode: "any" });
```

I could be wrong about it being a module that is calling the deprecated method... but it's the most likely situation. The warning message will have a stacktrace and that should indicate which script is calling the deprecated method.
